davinci-dome/src/DaVinci.ts

```diff
@@ -20,11 +20,7 @@ export interface Joint {
 
 export interface Bar {
     index: number
-    jointA: Joint
-    jointB: Joint
-    jointC: Joint
-    jointD: Joint
-    planeJoint?: Joint
+    joints: Joint[]
```

Comment:
jointA to jointD changed to joint[0] to joint[3], which allows for easily removing sawed off joints (below the plane)

Reply:
oke